web/src/main/webapp/resources/js/atsy-statehistory.js

```diff
@@ -10,6 +10,18 @@ function isNewState() {
   return queryStr.match(/state\=/);
 }
 
+function escapeXml(unsafe) {
```

Comment:
Yes there is a method for escaping html however it can only be used on data tables. Thats why i introduced this new method.